lnd/lnd.go

```diff
@@ -15,6 +15,10 @@ import (
 	"gopkg.in/macaroon.v2"
 )
 
+const (
+	MSAT_PER_SAT = 1000
```

Comment:
where is this used?

Reply:
Hmm, must have slipped in from my C-Lightning work, not used anywhere atm, deleted.